### PR TITLE
[F5 BigIP] Clarify the lack of support for syslog messages in README

### PR DESCRIPTION
**Pull request by kcreddy**
## Proposed commit message
* Clarify the lack of support for syslog messages in README

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

- Relates https://github.com/elastic/integrations/issues/7236

### CODE REVIEW

File: packages/f5_bigip/_dev/build/docs/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 The [F5 BIG-IP](https://www.f5.com/products/big-ip-services) integration allows users to monitor LTM, AFM, APM, ASM, and AVR activity. F5 BIG-IP covers software and hardware designed around application availability, access control, and security solutions.
 
-Use the F5 BIG-IP integration to collect and parse data from F5 BIG-IP using **telemetry streaming** and then visualize that data in Kibana.
+Use the F5 BIG-IP integration to collect and parse data from F5 BIG-IP using **telemetry streaming** and then visualize that data in Kibana. This integration currently only supports telemetry JSON-formatted data generated by following F5 features: LTM, AFM, APM, ASM, and AVR. This integration doesn't support ingesting or parsing the **syslog events** produced at the operating system level. Use the [Custom Logs integration](https://docs.elastic.co/integrations/log) to ingest these system-level log messages instead.
```
Comment (efd6) [inline packages/f5_bigip/_dev/build/docs/README.md:7]:
```suggestion
Use the F5 BIG-IP integration to collect and parse data from F5 BIG-IP using **telemetry streaming** and then visualize that data in Kibana. The integration currently only supports JSON-formatted telemetry data generated by following F5 features: LTM, AFM, APM, ASM, and AVR. Ingesting or parsing **syslog events** produced at the operating system level. Use the [Custom Logs integration](https://docs.elastic.co/integrations/log) to ingest these system-level log messages instead.
```

Comment (kcreddy) [reply packages/f5_bigip/_dev/build/docs/README.md:7]: Hey Dan, do you mean: 
Ingesting or parsing **syslog events** produced at the operating system level _is not supported_ ?

Comment (efd6) [reply packages/f5_bigip/_dev/build/docs/README.md:7]: Yes, thanks for interpreting my meaning. Indeed that is what I had intended.

Comment (kcreddy) [reply packages/f5_bigip/_dev/build/docs/README.md]: Thanks Dan, updated accordingly.

**Review by efd6 — APPROVED** [review]

**Pull request merged**